src/github/deployment.ts

```diff
@@ -68,10 +68,7 @@ export const processDeployment = async (
 		deploymentEnvironment: environment
 	}, "processing deployment message!");
 
-	const metrics = {
-		trigger: "deployment_queue"
-	};
-	const jiraPayload: JiraDeploymentBulkSubmitData | undefined = await transformDeployment(newGitHubClient, webhookPayload, jiraHost, "webhook", metrics, logger, gitHubAppId);
+	const jiraPayload: JiraDeploymentBulkSubmitData | undefined = await transformDeployment(newGitHubClient, webhookPayload, jiraHost, "webhook", logger, gitHubAppId);
```

Comment:
/ultranit not from your change but since we are here can we change this line to
`const jiraPayload = await transformDeployment(newGitHubClient, webhookPayload, jiraHost, "webhook", logger, gitHubAppId);`

dropping the type(since its inferred) just makes it way easier to read since its got that` | undefined `carry on

Reply:
lol ultranit

Reply:
ultra fixed.